integration-tests/README.md

```diff
@@ -70,6 +74,28 @@ The file must contain one property per line, the key must start with `druid_` an
 Note that when bringing up docker containers through mvn and -Doverride.config.path is provided, additional
 Druid routers for security group integration test (permissive tls, no client auth tls, custom check tls) will not be started.   
 
+### Troubleshooting
```

Comment:
nit: I would suggest putting this section much further down in the file, probably near the end